Auth/OAuth2/class.FlipsideAuthenticator.php

```diff
@@ -3,9 +3,16 @@
 
 class FlipsideAuthenticator extends OAuth2Authenticator
 {
+    /** An instance of the Settings class */
+    protected $settings;
+    /** The prfiles app URL */
+    var $profilesUrl;
+
     public function __construct($params)
     {
         parent::__construct($params);
+        $this->settings = \Settings::getInstance();
+        $this->profilesUrl = $this->settings->getGlobalSetting('profiles_url', 'https://profiles.burningflipside.com/');
```

Comment:
I'm still not a fan of this... On my dev system it actually points to profiles.burningflipside.com and uses the API to authenticate. I also have the profiles code base running there as well. With this change I have to pick either or...